src/main/java/com/bigstep/datalake/DLFileSystem.java

```diff
@@ -133,6 +148,7 @@
     private String homeDirectory;
     private short defaultFilePermissions;
     private short defaultUMask;
+    private boolean shouldUseEncryption = false;
```

Comment:
Typically you name this variable something like encryptionEnabled but it's not a problem right now.